_posts/2020-09-21-changelog-131.markdown

```diff
@@ -0,0 +1,95 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #131"
+date: 2020-09-21 12:00:00 +0300
+---
+
+
+## New Features
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6039 -->
+* Provide intention to create a method from an unresolved method call (by [@Kobzol])
+
+    {% include gif-img.html path="/assets/posts/changelog-131/create_method" w="364px" %}
```

Comment:
Maybe make all screenshots bigger (and set the same width). We usually use `700px`. What do you think?

Reply:
Thanks, good idea